setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = graspologic
-version = 2.0.1
+version = 3.0.0
```

Comment:
believe not having python 3.7 support warrants a major version bump?

Reply:
agreed, it drives me nuts that they don't do this for numpy/scipy when it clearly is a major change that will break your users